dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -1913,4 +1914,26 @@ public boolean isWorkflowInProgress () {
 		return null != this.getMap().get(Contentlet.WORKFLOW_IN_PROGRESS) &&
 				Boolean.TRUE.equals(this.getMap().get(Contentlet.WORKFLOW_IN_PROGRESS));
 	}
+
+	/**
+	 * Do a smart copy of the contentlet into the content argument
+	 * @return
+	 */
+	@JsonIgnore
+	public Contentlet copy (final Contentlet contentlet) {
+
+		if (null != contentlet && null != contentlet.map) {
+
+			if (this.loadedTags) {
+
+				contentlet.loadedTags = true;
```

Comment:
a test for this logic can be implemented

Reply:
this code is already tested by the transformer test, it is just a refactor.